packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -55,6 +55,14 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
       return;
     }
 
+    if (gltfMaterial.extensions &&
+        gltfMaterial.extensions['KHR_materials_pbrSpecularGlossiness']) {
+      console.warn(`Material ${gltfMaterial.name} uses a deprecated extension
+          "KHR_materials_pbrSpecularGlossiness", please use
+          "pbrMetallicRoughness" instead.`);
```

Comment:
Let's add a little detail here, like "Specular Glossiness materials are currently supported for rendering, but not for our scene-graph API, nor for auto-generation of USDZ for Quick Look."

Reply:
Done.